packages/components/src/navigation/README.md

```diff
@@ -1,26 +1,74 @@
 # Navigation
 
-Navigation is a component which renders a component which renders a heirarchy of menu items.
+Render a heirarchy of menu items into a waterfall style navigation.
```

Comment:
Great description.  One small typo ("heirarchy") and optional, but should we include "flat" to help signal the data structure used?

```suggestion
Render a flat array of menu items into a waterfall style hierarchy navigation.
```

I'm not sure if that makes things more clear or not; will leave that up to your discretion.